gcovr/formats/html/write.py

```diff
@@ -69,16 +69,23 @@ def reuse_value(*args):
     def __call__(self, *args):
         return self.get(*args)
 
+# html_theme string is <theme_directory>.<color> or only <color> (if only color the use default)
+# examples: github.green github.blue or blue or green
+def get_theme_name(html_theme):
```

Comment:
Add type annotations:
```suggestion
def get_theme_name(html_theme: str) -> str:
```